op-chain-ops/genesis/genesis.go

```diff
@@ -72,10 +76,13 @@ func NewL2Genesis(config *DeployConfig, l1StartHeader *types.Header) (*core.Gene
 		GraniteTime:                   config.GraniteTime(l1StartTime),
 		HoloceneTime:                  config.HoloceneTime(l1StartTime),
 		InteropTime:                   config.InteropTime(l1StartTime),
+		L2BlobTime:                    config.L2BlobTime(l1StartTime),
 		Optimism: &params.OptimismConfig{
 			EIP1559Denominator:       eip1559Denom,
 			EIP1559Elasticity:        eip1559Elasticity,
 			EIP1559DenominatorCanyon: &eip1559DenomCanyon,
+			IsSoulBackedByNative:     config.IsSoulBackedByNative,
```

Comment:
Do we have any tests for the case when IsSoulBackedByNative == false? Is there any possibility that we might use this setting on mainnet, or is it just a configuration we don’t intend to use?

Reply:
Probably will never use the `IsSoulBackedByNative == false` case, @qizhou , should we just remove this option?